tests/pw/pages/storeAppearancePage.ts

```
@@ -0,0 +1,113 @@
+import { BasePage } from '@pages/basePage';
+import { selector } from '@pages/selectors';
+import { helpers } from '@utils/helpers';
+import { data } from '@utils/testData';
+
+// selectors
+const singleStoreCustomer = selector.customer.cSingleStore;
+
+export class StoreAppearancePage extends BasePage {
+    constructor(page: Page) {
+        super(page);
+    }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)